openfecwebapp/templates/legal-enforcement-landing.html

```diff
@@ -0,0 +1,103 @@
+{% extends "layouts/legal-doc-landing.html" %}
+
+{% block content %}
+<header class="heading--main">
+  <h1>Enforcement matters</h1>
+</header>
+<div class="content__section">
```

Comment:
Instead of nesting `<p>` tags in `<li>` tags, let's just add `content__section--narrow` to this `div` like we do below. It will achieve the same effect.